lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1536,7 +1536,6 @@ fc_extras
                         standard_name = coord_name
                     else:
                         standard_name = None
-                        long_name = standard_name
```

Comment:
long_name is already None so this line doesn't do anything - so I removed to make this code mode clear